app/build.gradle

```diff
@@ -47,15 +47,16 @@ android {
 }
 
 dependencies {
+    implementation platform('androidx.compose:compose-bom:2023.01.00')
     implementation 'androidx.activity:activity-compose:1.6.1'
     implementation 'androidx.core:core-ktx:1.9.0'
-    implementation "androidx.compose.material:material:$compose_version"
-    implementation "androidx.compose.ui:ui:$compose_version"
-    implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
+    implementation "androidx.compose.material:material"
+    implementation "androidx.compose.ui:ui"
+    implementation "androidx.compose.ui:ui-tooling-preview"
     implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
     implementation "androidx.navigation:navigation-compose:$nav_version"
-    implementation "androidx.room:room-runtime:$room_version"
+    implementation "androidx.room:room-runtime"
```

Comment:
does this dependency get the versioning from BOM?

Reply:
I dont think so since it's not part of compose library

Reply:
It actually builds and runs...but it seems like it definitely shouldn't and I would think that was a mistake. I'm going to put it back in to be safe.